docs/developer-docs/security/security-best-practices/inter-canister-calls.mdx

```diff
@@ -269,11 +279,73 @@ mod test {
 }
 ```
 
+</TabItem>
+
+<TabItem value="motoko" label="Motoko">
+
+```motoko
+import Result "mo:base/Result";
+import Text "mo:base/Text";
```

Comment:
I eliminated this import (hashing `Principal`s directly),
see https://play.motoko.org/?tag=2353996190.

Contains other stylistic changes too, feel free to take or ignore those.